main.tf

```diff
@@ -74,7 +74,8 @@ resource "azurerm_app_service" "main" {
   "FLASK_ENV" = "developement"
   "LOAD_DISABLED" = ""
   "SECRET_KEY" = "real_key"
-  "Mongo_db" = "todo_db"
+  "LOG_LEVEL" = "DEBUG"
+  "Mongo_db" = "todoapp-cosmos-mongo-db"
```

Comment:
Would be nice to have Loggly set up for the deployed app as well, but this isn't required.